.travis.yml

```diff
@@ -32,7 +32,7 @@ script:
   # Test if installation works
   - bin/adminconsole sulu:build dev --no-interaction
   # Test if PHPUnit installation works
-  - SYMFONY_ENV=test bin/simple-phpunit
+  - SYMFONY_PHPUNIT_VERSION=5.7.27 bin/simple-phpunit
```

Comment:
Why do we need to set this version?

Reply:
current issue with the phpunit bridge after phpunit did remove the 5.7 branch: https://github.com/sebastianbergmann/phpunit/issues/3413

Reply:
Looks like this will be fixed in a future symfony version, right? This PR seems to solve this issue: https://github.com/symfony/symfony/pull/29265

Reply:
Do we still need this PR if the symfony PR is merged?

Reply:
@danrot undo that change but `SYMFONY_ENV` should be set in the `phpunit.xml.dist` to avoid problems locally

Reply:
@danrot tests will fail until the symfony fix is released thats why I did set it as last test in the travis.ci.